src/java.desktop/macosx/classes/com/apple/laf/AquaComboBoxButton.java

```diff
@@ -228,6 +230,17 @@ protected void doRendererPaint(final Graphics g, final ButtonModel buttonModel,
 
         rendererPane.paintComponent(g, c, this, left, top, cWidth, height, shouldValidate); // h - (insets.top + insets.bottom) );
 
+        // fix for 8283214
```

Comment:
we usually do not mention the bugid in the comments, it can be found in the git history, just describe why it is needed.

Reply:
Updated.